cli/cmd/scorecard.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/scorecard"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(scorecardCmd)
+
+	scorecardCmd.Flags().StringVar(&flags.scorecard.policyPath, "policy-path", "", "Path to directory containing validation policies")
+	scorecardCmd.MarkFlagRequired("policy-path")
+
+	scorecardCmd.Flags().StringVar(&flags.scorecard.targetProjectID, "project", "", "Project to analyze (conflicts with --organization)")
+
+	scorecardCmd.Flags().StringVar(&flags.scorecard.bucketName, "bucket", "", "GCS bucket name for storing inventory")
+	scorecardCmd.MarkFlagRequired("bucket")
+}
+
+// getEnvProjectID finds the implict environment project
+func getEnvProjectID() (string, error) {
+	project := os.Getenv("GOOGLE_PROJECT")
```

Comment:
I used this environment variable because it's already established with other tools (ex. Terraform) but in eb08fc6b80beff3a36307f99a359bf4daf36357e I updated it to accept a command-line flag as well (or leave both off entirely).